### DISCUSSION TITLE
How to re-install tinytex with quarto

### DISCUSSION THREAD

**Discussion by produnis** [quarto-dev/quarto-cli#2574] [Q&A] [answered]
Hi there, 
on my Linuxmachine, as a standard user, I typed in
`quarto install tool tinytex`
and everything went smooth, and I was able to render PDFs via
`quarto render MyPDF.qmd`

However, I dared to type in:
`sudo quarto install tool tinytex`
which also installed TinyTex and went through smoothly.
I am able to render PDFs via
`sudo quarto render MyPDF.qmd`

My Problem now is, that without sudo, I cannot render any longer as "normal" user
If I type in (without sudo)
`quarto render MyPDF.qmd`
I get the answer
```
No TeX installation was detected.

Please run 'quarto install tool tinytex' to install TinyTex.
If you prefer, you may install TexLive or another TeX distribution.
```

If I type in:
`quarto install tool tinytex`

I get the answer
`tinytex is already installed and up to date.`

So, how can I de- or re-install tinytex with quarto?

Thx in advance...

**Comment by cderv** [comment, marked as answer]
Hi. Which is your quarto version ? `quarto --version` 

We did some changes recently and behavior can differ depending of it. 

Can you find if `tlmgr` is in your PATH and which one ? 

@dragonstyle maybe a scenario here we need to look closer too. I thought it would not matter but maybe it is. Flagging that for you.

**Reply by produnis** [reply]
Hi, I am on `1.1.189`
and 
`which tlmgr` 
says
`which: no tlmgr in PATH`

**Reply by produnis** [reply]
I updated to `1.1.251`
but still same behaviour

**Reply by dragonstyle** [reply]
If you installed using `sudo`, I would try uninstalling using `sudo` as well:

`quarto remove tool tinytex`

Then try reinstalling without sudo. Starting in the later v1.2 builds of Quarto, we do smarter detection of TinyTex that is designed to address these types of problems, so once you successfully uninstall, perhaps you would try a 1.2 build?

https://quarto.org/docs/download/prerelease.html

**Reply by produnis** [reply]
Hi there, and thx a lot again. 

running
```
sudo quarto remove tool tinytex
quarto remove tool tinytex
quarto install tool tinytex
```
did the trick.
